shell/common/vsync_waiter_fallback.h

```diff
@@ -12,6 +12,7 @@
 
 namespace flutter {
 
+/// A fake |VsyncWaiter| that will fire at 60 fps.
```

Comment:
"... using a timer that has nothing to do system vsync". This is used by default on platforms that don't provide a system vsync waiter or when the system vsync waiter is temporarily disengaged.

Reply:
Done.